app/frontend/packs/nationalities-component.js

```diff
@@ -0,0 +1,118 @@
+const nationalitiesComponent = () => {
+  if (
+    document.querySelector(
+      "[for=candidate-interface-nationalities-form-other-nationality2-field-error]"
+    ) ||
+    document.querySelector(
+      "[for=candidate-interface-nationalities-form-other-nationality3-field-error]"
+    )
+  ) {
+    return;
+  }
+
+  addRemoveLink(
+    "candidate-interface-nationalities-form-other-nationality2-field"
+  );
+
+  addRemoveLink(
+    "candidate-interface-nationalities-form-other-nationality3-field"
+  );
+
+  addAddNationalityButton(
+    "#candidate-interface-nationalities-form-other-other-conditional"
+  );
+
+  hideSection(
+    "candidate-interface-nationalities-form-other-nationality2-field"
+  );
+
+  hideSection(
+    "candidate-interface-nationalities-form-other-nationality3-field"
+  );
+
+  function addRemoveLink(selector) {
+    const labelEl = document.querySelector(`[for=${selector}]`);
+    const removeLink = document.createElement("a");
+    removeLink.innerHTML = "Remove";
+    removeLink.classList.add("govuk-link", "remove-link");
+
+    // This has to be a link and not a button as the govuk-link class requires an
+    // href to apply  it's styling
+    removeLink.href = "#";
+    labelEl.appendChild(removeLink);
+
+    removeLink.addEventListener("click", function () {
+      addRemoveLinkEvent(labelEl, selector);
+    });
```

Comment:
```suggestion
    removeLink.addEventListener("click", (event) => {
      event.preventDefault();
      addRemoveLinkEvent(labelEl, selector);
    });
```

Reply:
Also, I prefer calling event handlers e.g. `handleRemoveLinkClick`, but that's more of a style suggestion.